apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -49,6 +49,22 @@ export function LoginForm({}: Props) {
     }
   };
 
+  const serverErrorString = useMemo<string>(() => {
+    return Array.isArray(error?.message) ? error?.message[0] : error?.message;
+  }, [error]);
+
+  const emailServerError = useMemo<string>(() => {
+    if (serverErrorString === 'User not found') return 'Account does not exist';
+    if (serverErrorString === 'email must be an email') return 'Please provide a valid email';
+
+    return '';
+  }, [serverErrorString]);
+
+  const passwordServerError = useMemo<string>(
+    () => (serverErrorString === 'Wrong credentials provided' ? 'Incorrect username or password' : ''),
```

Comment:
I don't think that the change to ```Incorrect username or password``` is correct here as there is
a) no login via username and
b) if the user has entered an invalid email there will be another error -> ```Account does not exist```

So we can surely say here that only the password entered is incorrect and not the combination of both. What do you think?

Reply:
Also, the tests are now failing because of the changes to the error message you commited 😅

Reply:
I see, it's an email there, So I think it will be better to update it to `Incorrect Email or Password` as in security we don't want the bad guy to know precisely that only password is incorrect. 
@scopsy any thoughts?

Reply:
We already send an error when there is no user found for a given email and "security through obscurity is no security" - many services show when there is no user found for an given email address as it helps the user to better understand what went wrong when trying to login

Reply:
Tend to aggree with @venarius, we can block enumeration with throttling and captchas to better handle it. Actually it was me who merged the suggestion, need to stop relying on GitHub's UI for those things. Sorry about that.